index.html

```diff
@@ -2120,588 +2138,378 @@ <h3>Thing Root Object</h3>
-  In that case the URI can be defined as template as defined in [[RFC6570]]:
-  <code>http://192.168.1.25/left{?p,d}</code>.
+  <span class="rfc2119-assertion" id="td-version-container">
+  The <code>version</code> <a>Map</a> MAY be used to provide additional application-
+  and/or device-specific version information based on terms from non-TD vocabularies.
+  </span>
+  See <a href="#sec-context-extensions"></a> for how to include additional namespaces.
   </p>
 
-  <p><span class="rfc2119-assertion" id="td-uriVariables-names">
-  In such a case, the URI Template variables MUST be collected
-  in the JSON-object based <code>uriVariables</code> member
-  with the associated (unique) variable names as JSON names.
+</section> <!-- end of id="version-json"-->
+
+<section id="security-serialization-json">
+<h3><code>securityDefinitions</code> and <code>security</code></h3>
+
+  <p>
+    In a <code>Thing</code> instance, the value assigned to
+    <code>securityDefinitions</code> is a <a>Map</a> of instances of
+    <code>SecurityScheme</code>.
+    <span class="rfc2119-assertion" id="td-security">
+    All name-value pairs of a <a>Map</a> of <code>SecurityScheme</code> instances
+    MUST be serialized as members of the JSON object that results from serializing the <a>Map</a>;
+    the name of a pair MUST be serialized as a JSON string and the value of the pair, an
+    instance of <code>SecurityScheme</code>, MUST be serialized as a JSON object.
   </span></p>
 
-  <p><span class="rfc2119-assertion" id="td-uriVariables-dataschema">
-  The serialization of each value in the map assigned to <code>uriVariables</code>
-  in an instance of <code>Form</code> MUST
-  rely on the <a>Class</a> <a href="#dataschema" class="sec-ref"><code>DataSchema</code></a>,
-  whose serialization is defined in <a href="#data-schema-serialization-json"></a>.
+  <p><span class="rfc2119-assertion" id="td-security-schemes">
+    All name-value pairs of an instance of one of the <a>Subclasses</a> of
+    <code>SecurityScheme</code>,
+    where the name is a <a>Vocabulary Term</a> included in the
+    <a>Signature</a> of that <a>Subclass</a> or in the <a>Signature</a> of
+    <code>SecurityScheme</code>, MUST be serialized as members of the JSON object
+    that results from serializing the <code>SecurityScheme</code> <a>Subclass</a>'s
+    instance, with the <a>Vocabulary Term</a> as name.
   </span></p>
 
-  <p>A TD snippet using a URI Template and <code>uriVariables</code> is given below:</p>
+  <p>
+  The following TD snippet shows a simple security configuration specifying
+  basic username/password authentication in the header.
+  The value given for <code>in</code> is actually the <a>Default Value</a> (<code>header</code>)
+  and could be omitted.
+  First, a named security configuration must be given
+  in the <code>securityDefinitions</code> map.
+  Second, that definition must be activated by inclduing its JSON name in the
```

Comment:
"inclduing" -> "including" (typo).   Note: First, Second, implies they need to be in order.   This is not actually the case: it would be acceptable to reverse the order.   Not nice, but a library might do so.   I would just remove "First, " and "Second, " from these sentences.

Reply:
FIxed.